pkg/cmd/cli/cmd/build.go

```diff
@@ -214,5 +214,6 @@ $ s2i build . centos/ruby-22-centos7 hello-world-app
 	buildCmd.Flags().VarP(&(cfg.RuntimeArtifacts), "runtime-artifact", "a", "Specify a file or directory to be copied from the builder to the runtime image")
 	buildCmd.Flags().StringVar(&(networkMode), "network", "", "Specify the default Docker Network name to be used in build process")
 	buildCmd.Flags().StringVarP(&(cfg.AsDockerfile), "as-dockerfile", "", "", "EXPERIMENTAL: Output a Dockerfile to this path instead of building a new image")
+	buildCmd.Flags().BoolVarP(&(cfg.KeepSymlinks), "keep-symlinks", "", false, "Copy symlinks as symlinks. Default behavior is to follow symlinks and copy files by content")
```

Comment:
this is only relevant when --copy is passed, right?

Reply:
(and other cases where we end up in the "copy, not clone" path)

Reply:
Yes, only with `--copy`. For completion, I tested old code 'git clone' path and that works well with symlinks, no change needed there.

Thanks for your comment, I added that information to `--keep-symlinks` flag description